src/components/AppHeader.vue

```diff
@@ -43,7 +43,7 @@
       </b-nav-item-dropdown>
       <b-nav-item
       v-if="!isAdmin"
-      to="/blog/index?page=1">
+      to="/blogs">
```

Comment:
Modified the path the user goes to. Once the frontend is connected to the backend, we can make this request to the backend and we can hide the query params from the user instead of placing it in the url by using a `currentPage` we hold in state and making changes that way/ The index does not need the query in the url since we are holding the currentPage in state

Reply:
i'm hesitant to switch to plural /blogs because it seems like an antipratice to have both singular and plural as potential path routes

Reply:
The reason I went with this was mainly to remove the `/index?page=1` from the URL. As a general practice this is not how we should handle the routing. Since we will be paginating the blog posts, that number in the URL will be changing. Instead of having to constantly change and re-access the URL, we will hold the `currentPage` in state and make a fetch to `/blog/index?page=<currentPage>` whenever we want to get more blog posts. That way the URL will not have to change and we will be able to track which page the user is viewing locally in state to determine what the next posts we are loading onto the page should be.

If not this then what else would you suggest @rancerun ? We could maybe use `/blog` instead? The main goal here was to remove the query at the end of the URL (and also index, we don't need the index deeper in the route either)

Reply:
it seems unintuitive for me to have pages but not have them display in the url. if i was a user and i wanted to return to a previous page - how would i be able to handle that?

in terms of naming convention, what do you think about /blog/page/:number